app/src/main/java/org/greenstand/android/TreeTracker/dashboard/DashboardViewModel.kt

```diff
@@ -70,60 +66,44 @@ class DashboardViewModel(
     private val syncObserver = Observer<List<WorkInfo>> { infoList ->
         when(infoList.map { it.state }.elementAtOrNull(0)) {
             State.BLOCKED -> {
-                if (_isSyncing != null) {
-                    showSnackBar?.invoke(R.string.sync_blocked)
-                }
-
+                showSnackBar?.invoke(R.string.sync_blocked)
                 _isSyncing = false
             }
             SUCCEEDED -> {
-                if (_isSyncing != null) {
+                if (_isSyncing != false) {
                     showSnackBar?.invoke(R.string.sync_successful)
                 }
 
                 _isSyncing = false
             }
             State.CANCELLED -> {
-
-                if (_isSyncing != null) {
-                    showSnackBar?.invoke(R.string.sync_stopped)
-                }
-
+                showSnackBar?.invoke(R.string.sync_stopped)
                 _isSyncing = false
             }
             State.FAILED -> {
-                if (_isSyncing != null) {
-                    showSnackBar?.invoke(R.string.sync_failed)
-                }
-
+                showSnackBar?.invoke(R.string.sync_failed)
                 _isSyncing = false
             }
             State.RUNNING -> {
-                if (_isSyncing != null) {
-                    showSnackBar?.invoke(R.string.sync_started)
-                }
-
+                showSnackBar?.invoke(R.string.sync_started)
                 _isSyncing = true
             }
             State.ENQUEUED -> {
-                if (_isSyncing != null) {
-                    showSnackBar?.invoke(R.string.sync_preparing)
-                }
-
+                showSnackBar?.invoke(R.string.sync_preparing)
                 _isSyncing = true
             }
         }
     }
 
     init {
-        updateData()
```

Comment:
This method was here to refresh the screen when going back to it after capturing trees. We still need it.